### ISSUE TITLE
numbering exercises?

### ISSUE THREAD

**Issue by LRDamsma** [LibraryCarpentry/lc-data-intro#198]
Hi all,

There are many exercises in this lesson. Would it be an idea to number them (as they are in episode 3) ? 

While acting as a helper for this lesson, there was much confusion in the break-out rooms about locating the right exercise, and sometimes people looking at different exercises thinking they were doing the same question.

**Comment by JRTieszen** [comment]
I agree with this sentiment. A problem that I noticed while looking through the lesson was in how many exercise questions there were without distinguishing signs beyond the text of the questions displayed. This problem occurs in multiple sections and can potentially confuse users attempting to go through the questions, especially if they are intended to be answered in groups like within the Regular Expressions section. The questions themselves are fairly distinct, but some form of numbered labeling would significantly reduce the confusion addressed above.
